WebGoatCore/Models/CreditCard.cs

```diff
@@ -182,9 +192,9 @@ private void WriteCreditCardFile(XDocument xmlDocument)
             {
                 CreateNewCreditCardFile();
             }
-            catch (Exception ex)
+            catch (Exception)
             {
-                throw ex;
+                throw;
             }
```

Comment:
Is there a need for this catch at all, if the exception is simply re-thrown? This is the default if no catch handler matches the exception.

Reply:
Done + removed other similar constructs.